pkg/kubelet/cm/container_manager.go

```diff
@@ -118,6 +118,12 @@ type ContainerManager interface {
 
 	// UpdateAllocatedDevices frees any Devices that are bound to terminated pods.
 	UpdateAllocatedDevices()
+
+	// GetAllDevices returns information about all the devices known to the manager
+	GetAllDevices() podresourcesapi.ResourceDeviceMap
+
+	// GetAllCPUs returns the assignable (not allocated) CPUs
+	GetAllCPUs() []int64
```

Comment:
should we just call this `GetAllocatableCPUs`?

Reply:
I see your point. The original idea was that from the manager perspective the API is returning all the resources - CPUs in this case. But being consistent all across the stack is probably better, so I'll change accordingly.